src/components/About.js

```diff
@@ -70,25 +70,25 @@ const About = ({ dispatch }) => (
               The Crime Data Explorer publishes nation-wide crime data collected
               by the FBI in a digital format. The tool allows you to view
               trends and download bulk data allowing you to get a better
-              understanding of crime across the country.
+              understanding of reported crime across the country.
             </p>
           </div>
           <div className="md-col md-col-3">
             <h3 className="mt-tiny mb2 fs-18 sm-fs-22">Available datasets</h3>
             <ul className="m0 p0 fs-14 sm-fs-16 left-bars">
               <li className="mb2">
-                <Term dispatch={dispatch} id="Summary Statistics (SRS)">
+                <Term dispatch={dispatch} id="Summary Reporting System (SRS)">
                   Summary Statistics (SRS)
```

Comment:
@RyanSibley Why is the text of the term still "Summary Statistics" if it looks up "Summary Reporting System (SRS)"?